_docs/setup/kubernetes/quick-start.md

```diff
@@ -187,7 +187,7 @@ curl -L https://git.io/getLatestIstio | sh -
 
 1. Change directory to istio package. For example, if the package is istio-{{site.data.istio.version}}
 ```bash
-cd istio-{{site.data.istio.version}}
+cd istio-0.7.1
```

Comment:
I forgot to add my comment here:
This would require maintenance prior to branching.  It is also inconsistent with line 188.  I know there are other places in the docs where we want to remove maintenance burden on version changes and we don't want to incrementally make the problem worse :).  @geeknoid had indicated in the issue tracker for this issue that the wording used "For example" was good enough (and I think was meant to solve this particular problem).